extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -225,7 +228,9 @@ export class ConsentUI {
     // Add to fixed layer
     this.baseInstance_.getViewport().addToFixedLayer(this.parent_);
     if (this.isCreatedIframe_) {
-      this.loadIframe_(isActionPromptTrigger).then(() => {
+      // Set this for every iframe request to show, no need to reset.
+      this.isActionPromptTrigger_ = isActionPromptTrigger;
```

Comment:
I don't recommend caching the value because isActionPromptTrigger is tied to the `show()` action, not to the consentUI class. It's possible that `#show()` is called twice with different `isActionPromptTrigger` value. Caching it to the class, could lead to the value been overridden before the promise is resolved.
This won't lead to a race condition only because the notificationUIManager makes sure no two UI can be shown at the same time.

Reply:
Well I understand that you don't want to pass the variable to nested functions. I'm fine either way. But please make it clear that such race condition won't happen.

Reply:
Ok, thanks for detailing your concern. I will add comments to explain that the race condition is not possible.